packages/attune/src/server/repo/index/sign.rs

```diff
@@ -651,8 +651,20 @@ pub async fn handler(
     // Commit the transaction. At this point, the transaction may abort because
     // of a concurrent index change. This should trigger the client to retry.
     //
-    // TODO(#84): Add special handling for aborts to signal the client to retry.
-    tx.commit().await.unwrap();
+    // Technically we should probably check for specific error codes,
```

Comment:
[no action] Huh, I can't even _find_ an error code that corresponds to commit rollbacks in the [docs](https://docs.rs/sqlx/latest/sqlx/enum.Error.html). Maybe it's `InvalidSavePointStatement`? Well, it'll definitely come out when we run #106 against it.